kolibri/utils/cli.py

```diff
@@ -215,6 +215,14 @@ def initialize(debug=False):
             )
             update()
 
+def migrate_databases():
```

Comment:
Would suggest to add an underscore in front here -- to indicate that it's not API to be called externally.